
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="UTF-8">
<link rel="stylesheet" type="text/css" href="css/GENEDBQuery.css"/>
<link rel="stylesheet" type="text/css" href="css/IndividualResult.css"/>
<link type="text/css" href="css/GENEDBQuery_old.css" rel="stylesheet" />
<link type="text/css" href="css/styleFoot.css" rel="stylesheet" />
<title>IMGT/GENE-DB Consultation</TITLE>
</head>
<body id="GENE-DB">
<table style="width:100%;">
<tr>
<td align="left" nowrap valign="middle" width="70%">
<font size="+3" color="#000099"><b>THANK YOU</b><br/>
<b>for using <a href="/genedb/doc">IMGT/GENE-DB</a></b></font>
</td>
<td class="notshow" align=right valign=top width="30%"><a href="/"><img SRC="images/logo_IMGT2.png" ALT="WELCOME !" class="mainlogo"/></a>
</td>
</tr>
<tr>
<td class="logolegendleft">IMGT&reg;, the international ImMunoGeneTics information system&reg;</td>
<td class="logolegend">http://www.imgt.org</td>
</tr>
</table>
<hr/>
<img src="images/logoGene-DB.png" ALT="Gene-DB logo" class="mainlogo" />

<table class="citation">
<tr>
<td class="citation">
<b>Citing IMGT/GENE-DB</b>: Giudicelli, V. et al. Nucleic Acids Res., 33: D256 - D261 (2005).
PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/15608191/">15608191</a>
<a href="/PDF/NAR/33_D256-D261_2005.pdf"><img src="/images/pdficon.png" alt="pdf" /></a>
</td>
</tr>
</table>
<p class="programversion">IMGT/GENE-DB program version: <a href="/IMGTgenedbdoc/programversions.html">3.1.30</a> (18 September 2020)<br/>
<a href="/IMGTgenedbdoc/dataupdates.html">IMGT/GENE-DB data updates </a>
</p>

<h2 class="subtitle"><a href="/IMGTScientificChart/SequenceDescription/IMGTreferencesequences.html" >IMGT/GENE-DB reference sequences</a> in FASTA format:</h2>
<h2 class="subtitle">Amino acids  sequences with gaps according to the <a href="/IMGTScientificChart/Numbering/IMGTnumberingCR.html">IMGT unique numbering</a><br /> for <span class="casimir">F+ORF+in-frame P with IMGT Gaps</span> alleles including orphons</h2>
<hr>
<pre>
The FASTA header contains 15 fields separated by '|':

1. IMGT/LIGM-DB accession number(s)
2. IMGT gene and allele name
3. species
4. IMGT allele functionality
5. exon(s), region name(s), or extracted label(s)
6. start and end positions in the IMGT/LIGM-DB accession number(s)
7. number of nucleotides in the IMGT/LIGM-DB accession number(s)
8. codon start, or 'NR' (not relevant) for non coding labels
9. +n: number of nucleotides (nt) added in 5' compared to the corresponding label extracted from IMGT/LIGM-DB
10. +n or -n: number of nucleotides (nt) added or removed in 3' compared to the corresponding label extracted from IMGT/LIGM-DB
11. +n, -n, and/or nS: number of added, deleted, and/or substituted nucleotides to correct sequencing errors, or 'not corrected' if non corrected sequencing errors
12. number of amino acids (AA): this field indicates that the sequence is in amino acids
13. number of characters in the sequence: nt (or AA)+IMGT gaps=total
14. partial (if it is)
15. reverse complementary (if it is)
</pre>
<hr/>
<b>Number of results = 103</b><br />
<pre>
>NW_001099007|IGKV1-10*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPSSLSASVGDRVTITCRASQGI......SNDLNWYQQKPGKAPKLLIYAA...
....SSLESEVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQPFKNYL
>NW_001099007|IGKV1-11*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQLTQSPSSLSASVGDRVTITCRASQGI......SSYLAWYQQKPGKAPKLLIYKA...
....SSLQSGVP.SRFSGSG..SGTEFTLTISSLQPEDFAVYYCQQRNSYP
>NW_001099007|IGKV1-12*01|Macaca mulatta|ORF|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPSSLSASVGDRVTITCRASQGI......SSYLAWYQQKPGKAPKPLIYYA...
....SNLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQYNSDP
>NW_001099007|IGKV1-13*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPSSLSASVGDRVTITCRASQGI......SDYLSWYQQKPGKAPKRLIYAA...
....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFAAYYCLQGYSTP
>NW_001099007|IGKV1-14*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPSSLSASVGDKVTITCRASQGI......SNALAWYQQKPGKAPKLLIYAA...
....SNLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFAVYYCQQRNSYP
>NW_001099007|IGKV1-15*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMSQSPSSLSASVGDRVTITCRASQGI......SSYLNWYQQKPGKAPKLLIYYA...
....NSLASRVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQGNSNP
>NW_001099007|IGKV1-16*01|Macaca mulatta|P|V-REGION|296932..297218|287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIKMT*SPFFLSASVGNRVTITFQATEGI......SNDLPWYQQKPGKSPKLFLYDA...
....KDLHTGVS.SRFGGSG..SGTDFTHTISSLEPEDFATYYSKQEFSHS
>NW_001099007|IGKV1-17*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPSSLSASVGDTVTITCRASQGI......SSYLNWFQQKPGKAPKLLIYAA...
....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCLQYKSYP
>NW_001099007|IGKV1-18*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPSSLSASVGDRVTITCQASQGI......SSWLAWYQQKPGKAPKLLLYKA...
....PGLQSGVP.SMFSGSG..SGTDFTLTISSLQPEYFATYYCQQFSSAL
>NW_001099007|IGKV1-19*01|Macaca mulatta|ORF|V-REGION|331269..331555|287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPCFLSASVGDTVAITCQASGGI......SNNLAWYQQKPGKSPKLFLYDA...
....KDLCTGVS.SKFSGRG..SGTDFTLTIRSLELEDFVAYYCEQKFSYP
>NW_001099007|IGKV1-2*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
AIQMTQSPSSLSASVGDTVTITCRASQSI......GSNLAWYQQKPGKVPKLLIYAA...
....STLQSEVP.SRFSGSG..SGTDFTLTISSLQPEEVATYYCQKCDSAP
>NW_001099007|IGKV1-20*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPSSLSASVGDRVTITCRASQGI......SSYLAWYQQKPGKAPKLLIYAA...
....SSLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQYNSNP
>NW_001099007|IGKV1-21*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPSSLSASVGDKVTITCHASQGI......SSWLAWYQQKPGKAPKPLIYAA...
....SSLQSGVP.SRFSGSG..SGTDYTLTISSLQPEDFATYYCQQYDDLP
>NW_001099007|IGKV1-22*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPSSLSASVGDKVTITCQASQSI......SSWLAWYQQKPGKAPKPLIYKA...
....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQYNSAP
>NW_001099007|IGKV1-23*01|Macaca mulatta|F|V-REGION| |224 nt|1| | ||74 AA|74+37=111|partial in 5'| |
.....................ACRASQSI......HKGLAWYQQKPGKAPKLLISYA...
....SNLYTDVP.SRFSGSG..SGTDFTLTISSLQPEDAATYYCFQYNSYP
>NW_001099007|IGKV1-3*01|Macaca mulatta|P|V-REGION| |283 nt|1| | | |94 AA|94+16=110| |rev-compl|
DFQMIQSPSSLTTSLGDRVTITCWASQGI......CNYLS*YHQKLGNPPQLLIYAA...
....PSLQSGVP.SRFSSSR..SGTHFTLTISSLQPEDFITYCCLQTYSS
>NW_001099007|IGKV1-5*01|Macaca mulatta|F|V-REGION| |287 nt|1| | ||95 AA|95+16=111| |rev-compl|
DIQMTQAPSFLSASVGDRVTITCQASQSI......SNDLAWYQQKPGKSPTLLIYDA...
....TNLHTGVP.SRFSGSG..SGADYSLTISSLELEDFAAYYCQQEYSYP
>NW_001099007|IGKV1-8*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPSSLSASVGDRVTITCRASQTI......SSYLAWYQQKPGKVPKLLIYAA...
....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSHP
>NW_001099007|IGKV1-9*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIQMTQSPSSLSASVGDRVTITCRASQGI......STYLNWYQQKPGKAPKRLIYAA...
....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCLQYNSDP
>AY963709|IGKV1S1*01|Macaca mulatta|F|V-REGION|165..451|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQGI......SSYLNWYQQKPGKAPKLLIYYA...
....NRLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQYNSLP
>AY963718|IGKV1S10*01|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQTI......SSYLAWYQQKPGKVPKLLIYAA...
....STLQSGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSHP
>AY963719|IGKV1S11*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDTVTITCRASQGI......SSWLAWYQQKPGKAPKLLIYAA...
....SSLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFAVYYCQQRNSYP
>AY963720|IGKV1S12*01|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQTI......SSYLAWYQQKPGKVPKLLIYAA...
....STLQSGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCLQYNSNP
>AY963721|IGKV1S13*01|Macaca mulatta|F|V-REGION|167..453|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDTVTITCRASQGI......SNYLAWYQQKPGKAPKLLIYAA...
....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFAAYYCLQHNSYP
>AY963722|IGKV1S14*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDTVTITCRASQGI......SNYLAWYQQKPGKAPKPLIYYA...
....SNLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQHNSYP
>AY963723|IGKV1S15*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDTVTITCRASQGI......SNYLAWYQQKPGKAPKPLIYYA...
....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQGNSNP
>AY963724|IGKV1S16*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDTVTITCRASQGI......SNYLAWYQQKPGKAPKPLIYYA...
....SSLESGVP.SRFSGSG..YGTDFTLTISSLQPEDFATYYCQQHNSYP
>AY963725|IGKV1S17*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQGI......SNNLAWYQQKPGKAPKLLIYAA...
....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCLQYNSNP
>AY963727|IGKV1S19*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDTVTITCRASQGI......SNYLNWYQQKPGKAPKLLIYAA...
....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFAAYYCLQHNSYP
>AY963710|IGKV1S2*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQGI......SNWLAWYQQKPGKAPKLLIYRA...
....SNLETGVP.SRFSGSG..SGTDFTLTISSLQPEDIATYYCQQHDNSP
>AY963728|IGKV1S20*01|Macaca mulatta|F|V-REGION|169..455|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQGI......STYLNWYQQKPGKAPKLLIYAA...
....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCLQYNSNP
>AY963729|IGKV1S21*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQGI......SNYLNWYQQKPGKAPKRLIYDA...
....SSLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFAAYYCLQYNSAP
>AY963730|IGKV1S22*01|Macaca mulatta|F|V-REGION|167..453|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDKVTITCRASQGI......SNALAWYQQKPGKAPKLLIYAA...
....SSLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQHNSNP
>AY963731|IGKV1S23*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQGI......SDYLSWYQQKPGKAPKLLIYAA...
....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFAAYYCLQGYSTP
>AY963732|IGKV1S24*01|Macaca mulatta|F|V-REGION|167..453|287 nt|1| | | |95 AA|95+16=111| | |
DIQMSQSPSSLSASVGDRVTITCRASQGI......SSYLNWYQQKPGKAPKLLIYYA...
....NSLASGVP.SRFSGSG..SGTEFTLTISSLQPEDFAAYYCLQGYSTP
>AY963733|IGKV1S25*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQGI......SSWLAWYQQKPGKVPKLLIYAA...
....STLQSGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSHP
>AY963734|IGKV1S26*01|Macaca mulatta|F|V-REGION|167..453|287 nt|1| | | |95 AA|95+16=111| | |
DIQMSQSPSSLSASVGDRVTITCRASQGI......SSYLNWYQQKPGKAPKLLIYYA...
....NSLASGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQGNSNP
>AY963736|IGKV1S27*01|Macaca mulatta|F|V-REGION|166..452|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQGI......SNWLAWYQQKPGKAPKLLIYAA...
....SNLETGVP.SRFSGSG..SGTDFTLTISSLQPEDIATYYCQQHDNSP
>AY963737|IGKV1S28*01|Macaca mulatta|F|V-REGION|165..451|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQGI......SSYLNWYQQKPGKAPKLLIYYA...
....NRLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQYNSLP
>AY963711|IGKV1S3*01|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDTVTITCRASQGI......SNYLAWYQQKPGKAPKPLIYYA...
....SSLESGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQHNSYP
>AY963712|IGKV1S4*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCQASQGI......SSWLAWYQQKPGKAPKLLLYKA...
....PGLQSGVP.SMFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSYP
>AY963713|IGKV1S5*01|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQTI......SSYLAWYQQKPGKAPKRLIYAA...
....SSLESGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSHP
>AY963714|IGKV1S6*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDKVTITCRASQGI......SSWLAWYQQKPGKAPKLLIYKA...
....SSLASGVP.SRFSGSG..SGTEFTLTISSLQPEDFATYYCQQHNSYP
>AY963715|IGKV1S7*01|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDTVTITCRASQGI......SSYLNWFQQKPGKAPKLLIYAA...
....TTLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFAAYYCLQHNSYP
>AY963716|IGKV1S8*01|Macaca mulatta|F|V-REGION|159..445|287 nt|1| | | |95 AA|95+16=111| | |
DIQMTQSPSSLSASVGDRVTITCRASQTI......SSYLAWYQQKPGKVPKLLIYAA...
....STLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQYNSAP
>AY963717|IGKV1S9*01|Macaca mulatta|F|V-REGION|168..454|287 nt|1| | | |95 AA|95+16=111| | |
DIQMSQSPSSLSASVGDTVTITCRASQGI......SNYLNWFQQKPGKAPKLLIYAA...
....TTLQSGVP.SRFSGSG..SGTDFTLTISSLQPEDFATYYCQQHNSYP
>NW_001099007|IGKV2-1*01|Macaca mulatta|F|V-REGION| |302 nt|1| | | |100 AA|100+11=111| |rev-compl|
DVVMTQSPLSLPITPGQPASISCRSSQSFVHS.DGNTYLSWYQQKPGQPPRLLIYKV...
....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCGQGTKVP
>NW_001099007|IGKV2-10*01|Macaca mulatta|P|V-REGION| |301 nt|1| | ||100 AA|100+11=111| |rev-compl|
DAVMTQLPLCLPVTLGELASVSYRLSHSPLHS.NGYTYLK*YLQKPGQPPRLLICLV...
....SNWDPGVS.DRFNGSV..SVTDFMLKITRVDIEDIGVYYCQQSTH*R
>NW_001099007|IGKV2-11*01|Macaca mulatta|ORF|V-REGION| |302 nt|1| | ||100 AA|100+11=111| |rev-compl|
DAVMSQPPLSLPVTPGEPGSISYRSSHSPLHS.NGCTFLNWDLQKPGQPPRLLIYLI...
....SNRDPGVP.DTFSGRE..SGTDFMLKISRVEAEDVGVYYCQQNTFFR
>NW_001099007|IGKV2-14*01|Macaca mulatta|F|V-REGION| |302 nt|1| | | |100 AA|100+11=111| |rev-compl|
DIVMIQNPLSLPVPPGQPASISCRSSQSFLHS.DESTYLDWYLQKPGQSPWLLIYLI...
....SHKFYGVP.SEFSGSR..SGTGFTPKFSKVEAEDVGVYCCEQGLQGP
>NW_001099007|IGKV2-15*01|Macaca mulatta|P|V-REGION| |296 nt|1| | ||98 AA|98+13=111| | |
DTVMTQTPLSLPVT.G.PASISCRSSQSLPYG.NGVNYLNWYLQKPDQPPQLLIYLG...
....SSRFPGVP.DRFTVSR..SDTDFTLQISRVKAEDVGVYY*VQCLQLP
>NW_001099007|IGKV2-3*01|Macaca mulatta|F|V-REGION| |302 nt|1| | | |100 AA|100+11=111| |rev-compl|
DIVMTQTPLSLPVTLGEPASISCRSSQSLLSS.NGYNYLNWYLQKPGQSPQLLIYYG...
....SNRASGVP.DRFSGSG..SGTDFTLKISRVEAEDVGVYYCMQALQTP
>NW_001099007|IGKV2-4*01|Macaca mulatta|P|V-REGION| |302 nt|1| | | |100 AA|100+11=111| |rev-compl|
DVAMTQSPLSLPVTPGQPASISCRSSQSLLHS.NGNTYLS*FQQKPGQSPRRLIYKV...
....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCMEGTHHP
>NW_001099007|IGKV2-6*01|Macaca mulatta|P|V-REGION| |302 nt|1| | | |100 AA|100+11=111| |rev-compl|
DIELTQTPHSLPVNPGEPASISC*SSQSLLHS.DGKTYFYWFLQKPGQHPRHPIYWV...
....SNWNSGAP.DRFSGSG..AGTDFTLRIGVVEAEDVGVYYYMQVTHWP
>NW_001099007|IGKV2-7*01|Macaca mulatta|P|V-REGION|183637..183932|302 nt|1| | | |100 AA|100+11=111| |rev-compl|
DIVMIQTPLSLPVTPGEPASISCRSNQNLLHS.DGNTYLHWYLQKPGQSPQLLIYGG...
....SNRFAGVP.DRFSGSG..SGTDFTLKISQVEAEDVGVY*CTQGIQFP
>NW_001099007|IGKV2-8*01|Macaca mulatta|P|V-REGION| |302 nt|1| | | |100 AA|100+11=111| |rev-compl|
DTVMTQPPLSLPITTGEPGSISFRPSHSPLHS.NGYTYLNWDLQNPGQPPWLPIYLV...
....SNRNPGVP.KRFRDSE..SMTTFRFKIRRMDAEDVEVYCCCQQSTHY
>NW_001099007|IGKV2-9*01|Macaca mulatta|P|V-REGION| |302 nt|1| | | |100 AA|100+11=111| |rev-compl|
AVVMTQPPLSLPVTPEEPYSISCRPSHSPLHS.NGYTYLNWVLQKPGQPPWLPIYWV...
....SNREPGVP.ERFSGSE..SVINFRLKIRRMDAEDVEVYCCQQSTHYS
>AY963738|IGKV2S1*01|Macaca mulatta|F|V-REGION|435..730|296 nt|1| | | |98 AA|98+13=111| | |
DTVMTQTPLSLPVT.G.PASISCRSSQSLPYG.NGVNYLNWYLQKPDQPPQLLIYLG...
....SSRFPGVP.DRFTVSR..SDTDFTLQISRVKAEDVGVYYCVQCLQLP
>AY963748|IGKV2S10*01|Macaca mulatta|F|V-REGION|401..702|302 nt|1| | | |100 AA|100+11=111| | |
DIVMTQTPLSLPITPGEPASISCRSSQSLLHS.NGNTYLHWYLQKPGQSPQLLIYGG...
....SNRASGVP.DRFSGSG..SGTDFTLKISKVEAEDVGVYYCVQAIAFP
>AY963749|IGKV2S11*01|Macaca mulatta|F|V-REGION|401..702|302 nt|1| | | |100 AA|100+11=111| | |
DIVMTQTPLSLPITPGEPASISCRSSQSLLHS.NGNTYLHWYLQKPGQSPQLLIYGG...
....SNRASGVP.DRFSGSG..SGTDFTLKISKVEAEDVGVYYCVHAIAFP
>AY963750|IGKV2S12*01|Macaca mulatta|F|V-REGION|400..701|302 nt|1| | | |100 AA|100+11=111| | |
DIVMTQTPLSLPVTPGEPASISCRSSQSLLHT.DGYTYLDWYLQKPGQSPQLLIYGG...
....SNRASGVP.DRFSGSG..SGTDFTLKISKVEAEDVGVYYCMQHKALP
>AY963751|IGKV2S13*01|Macaca mulatta|F|V-REGION|400..701|302 nt|1| | | |100 AA|100+11=111| | |
DIVMTQTPLSLPVTPGEPASISCRSSQSLLHT.DGYTYLDWYLQKPGQSPQLLIYGG...
....SNRASGVP.DRFSGSG..SGTDFTLKISKVEAEDVGVYYCMQHKALP
>AY963753|IGKV2S14*01|Macaca mulatta|ORF|V-REGION|415..719|305 nt|1| | ||101 AA|101+10=111| | |
DIVMTQTPLSLPVTPGEPASISRRSSQSLLDSEDGNTYLDWYLQKPGQSPQPLIYEV...
....SNRASGVP.DRFSGSG..SDTDFTLKISRVEAEDVGVYYCMQYTHIP
>AY963754|IGKV2S15*03|Macaca mulatta|F|V-REGION|404..705|302 nt|1| | | |100 AA|100+11=111| | |
DIVMTQTPLSLSVTPGQPASISCKSSQSLLHS.DGKTYLYWYLQKPGQSPQLLIYEV...
....SSRFSGVP.DRFSGSG..SGTDFTLKISRVEAEDVGVYYCMQGIHLP
>AY963755|IGKV2S16*01|Macaca mulatta|ORF|V-REGION|415..719|305 nt|1| | ||101 AA|101+10=111| | |
DIVMTQTPLSLPVTPGEPASISCRSSQSLLDSEDGNTYLDRYLQKPGQSPQLLIYEV...
....SNRASGVP.DRFSGSG..SDTDFTLKISRVEAEDVGVYYCMQGIEYP
>AY963756|IGKV2S17*01|Macaca mulatta|F|V-REGION|412..716|305 nt|1| | ||101 AA|101+10=111| | |
DIVMTQTPLSLPVTPGEPASISCRSSQSLLDSEDGNTYLDWYLQKPGQSPQLLIYEV...
....SNRASGVP.DRFSGSG..SDTDFTLKISRVEAEDVGVYYCMQALEFP
>AY963757|IGKV2S18*01|Macaca mulatta|F|V-REGION|424..725|302 nt|1| | | |100 AA|100+11=111| | |
DIVMIQTPLSLPVTLGEPASISCRSSQSLVYS.DGKTYLYWYLQKPGQSPQLLMYLV...
....SKRASGVP.DKFSGSG..SGTDFTLKISRVEAEDVGVYYCMQALRSP
>AY963759|IGKV2S19*01|Macaca mulatta|F|V-REGION|425..726|302 nt|1| | | |100 AA|100+11=111| | |
DIVMIQTPLSLPVTLGEPASISCRSSQSLVYS.DGKTYLYWYLQKPGQSPQLLMYLV...
....SKRASGVP.DKFSGSG..SGTDFTLKISRVEAEDVGVYYCMQALRSP
>AY963739|IGKV2S2*01|Macaca mulatta|F|V-REGION|442..742|301 nt|1| | | |100 AA|100+11=111| | |
DIVMTQTPLSLPVTPGEPASISCRSSQSLLHS.NGNTYLDWYLQKPGQSPRLLIYKV...
....TNRESGVP.DRFSGSG..SGTDFTLKISRVEPEDVGVYYCMQSTKDP
>AY963760|IGKV2S20*01|Macaca mulatta|F|V-REGION|415..719|305 nt|1| | ||101 AA|101+10=111| | |
DIVMTQTPLSLPVTPGEPASISCRSSQSLLDSEDGNTYLEWYLQKPGQSPQPLIYEV...
....SNRASGVP.DRFSGSG..SDTDFTLKISRVEAEDVGVYYCMQGIEYP
>AY963740|IGKV2S3*01|Macaca mulatta|F|V-REGION|442..742|301 nt|1| | | |100 AA|100+11=111| | |
DIVMTQTPLSLPVTPGEPASISCRSSQSLLHS.NGNTYLHWYLQKPGQSPRLLIYKV...
....TNRESGVP.DRFSGSG..SGTDFTLKISRVEPEDVGVYYCMQSTKDP
>AY963742|IGKV2S4*01|Macaca mulatta|F|V-REGION|466..766|301 nt|1| | | |100 AA|100+11=111| | |
DVVMTQSPLSLPITPGQPASISCRSSQSLVHS.DGNTYLSWYQQKPGQPPRLLIYKV...
....SNRYSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCGQGTHCL
>AY963743|IGKV2S5*01|Macaca mulatta|F|V-REGION|464..765|302 nt|1| | | |100 AA|100+11=111| | |
DVAMTQSPLSLPVTPGQPASISCRSSQSLLHS.NGNTYLSWFQQKPGQSPRRLIYKV...
....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCMEGTHHP
>AY963745|IGKV2S7*01|Macaca mulatta|P|V-REGION|453..754|302 nt|1| | | |100 AA|100+11=111| | |
DIVMIQTPLSLPVTPGEPASISCRSNQNLLHS.DGNTYLHWYLQKPGQSPQLLIYGG...
....SNRFAGVP.DRFSGSG..SGTDFTLKISQVEAEDVGVY*CTQGIQFP
>AY963746|IGKV2S8*01|Macaca mulatta|F|V-REGION|467..767|301 nt|1| | | |100 AA|100+11=111| | |
DVVMTQSPLSLPVTPGQPASISCRSSQSLVHS.DGKTYLNWLQQKPGQPPRRLIYQV...
....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCGQGTHVL
>AY963747|IGKV2S9*01|Macaca mulatta|F|V-REGION|467..767|301 nt|1| | | |100 AA|100+11=111| | |
DVVMTQSPLSLPVTPGQPASISCRSSQSLVHS.DGKTYLNWLQQKPGQPPRRLIYQV...
....SNRDSGVP.DRFSGSG..AGTDFTLKISRVEAEDVGVYYCVQGTHCL
>NW_001099007|IGKV3-1*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
QVILTQSPATLSLSPGERATLSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA...
....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDFAVYYCQKYSSSP
>NW_001099007|IGKV3-2*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
EIVMTQSPATLSLSPGERATLSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA...
....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDFAVYYCQQYSNWP
>NW_001099007|IGKV3-3*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
EIVMTQSPATLSLSPGETATLSCRASESV......GSYLAWYQQKPGQAPKLLVHSA...
....YFRATGIP.DRFSGSG..SRTEFTLTISSLEPEDVGVYHCQQYNDLL
>NW_001099007|IGKV3-4*01|Macaca mulatta|P|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
EIVMTQSPATLSLSPGERATLSCRASQSV......SSNLAWYQQKPGQAPRLLIYDA...
....SNRATGIP.DRFSGSG..SGTDFTLTISSLEPEDVGVYYCQQESNWP
>NW_001099007|IGKV3-5*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
EIVMTQSPATLSLSPGETATISCRTSQSV......SSYLAWYQQKPGQAPRLLIYGA...
....SSRATGIP.DRFSGSG..SGTDFTLTISSLEPEDFAVYYCQETSNLS
>NW_001099007|IGKV3-6*01|Macaca mulatta|P|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
EIVMTQSPATLSLSPGERATLSCRASQSV......SSSLAWYQQKPGQAPKLLIYGA...
....SSRATGIP.DGFSGSG..SGTEFTLTISSLEPEDVGVYYCQQDYSWP
>NW_001099007|IGKV3-7*01|Macaca mulatta|ORF|V-REGION| |270 nt|1| | ||90 AA|90+16=106|partial in 3'|rev-compl|
EIVMTQSPATLSLSPGERATLSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA...
....SSRVTGIP.DRFSGSG..SGTDFTLTISSLEPEDVAVYYCQQ
>NW_001099007|IGKV3-8*01|Macaca mulatta|P|V-REGION| |290 nt|1| | ||96 AA|96+15=111| |rev-compl|
EIVLTQSPTSMAVSQGERVTISCTASSSVS.....TSYLHWYQQKPGFPPRLLVYRT...
....SSLASGVP.ARFSGSG..SGTSYTLTISSMEAEDAANYYCQQENSIP
>NW_001099007|IGKV3-9*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
QVILTQSPATLSLSPGERATLSCRASQSV......SSYLAWYQQKPGQAPRLLIHSA...
....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDVGVYHCYQYYSGY
>AY963762|IGKV3S1*01|Macaca mulatta|F|V-REGION|205..491|287 nt|1| | | |95 AA|95+16=111| | |
EIVMTQSPATLSLSPGETATLSCRASESV......GSYLAWYQQKPGQAPKLLVHSA...
....YFRATGIP.DRFSGSG..SRTEFTLTISSLEPEDVGVYHCQQYNDLL
>AY963771|IGKV3S10*01|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+16=111| | |
EIVMTQSPATLSLSPRERATLSCRASQSV......SSNLAWYQQKPGQAPRLLIYYA...
....SNRATGIP.DRFSGSG..SGTDFTLTISSLEPEDVGVYYCQQESNWS
>AY963772|IGKV3S11*01|Macaca mulatta|F|V-REGION|234..520|287 nt|1| | | |95 AA|95+16=111| | |
QVILTQSPATLSLSPGERATLSCRASQSV......GSNLAWYQQKPGQAPRLLIYGA...
....SSRATGIP.DRFSGSG..SGTDFTLTISSLEPEDVAVYYCLQRSNWP
>AY963763|IGKV3S2*01|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+16=111| | |
EIVMTQSPATLSLSPGERATLSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA...
....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDVAVYYCLQRSNWP
>AY963764|IGKV3S3*01|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+16=111| | |
EIVMTQSPATLSLSPGERATLSCRASQSV......GSSLAWYQQKPGQAPRLLIYGA...
....SSRATGIP.DRFSGSG..SGTDFTLTISSLEPEDVAVYYCLQRSNWP
>AY963765|IGKV3S4*01|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+16=111| | |
EIVMTQSPATLSLSPGERATLSCRASQSV......SSRLAWYQQKPGQAPRLLIYDA...
....SSRVTGIP.DRFSGSG..SGTDFTLTISSLEPEDVAVYFCQQESNWS
>AY963766|IGKV3S5*01|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | ||95 AA|95+16=111| | |
EIVMTQSPATLSLSPGERATLSCRASQSV......SSRLAWYKQKPGQAPRLLIYDA...
....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDVAVYLCQQESNWS
>AY963767|IGKV3S6*01|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+16=111| | |
EIVMTQSPATLSLSPGERATLSCRASQSV......SSRLAWYQQKPGQAPRLLIYDA...
....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDVAVYFCQQESNWS
>AY963768|IGKV3S7*01|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+16=111| | |
EIVMTQSPATLSLSPGERATLSCRASQSV......SSSLAWYQQKPGQAPRLLIYGA...
....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDFAVYYCQQYSNWP
>AY963769|IGKV3S8*01|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+16=111| | |
EIVMTQSPATLSLSPRERATLSCRASQSV......SSNLAWYQQKPGQAPRLLIYYA...
....SNRATGIP.DRFSGSG..SGTDFTLTISSLEPEDVGVYYCQQESNWP
>AY963770|IGKV3S9*01|Macaca mulatta|F|V-REGION|206..492|287 nt|1| | | |95 AA|95+16=111| | |
EIVMTQSPATLSLSPGERATLSCRASQSV......SSYVAWYQQKPGQAPRLLIYGA...
....SSRATGIP.DRFSGSG..SGTEFTLTISSLEPEDFAVYYCQQYSNWP
>NW_001099007|IGKV4-1*01|Macaca mulatta|P|V-REGION| |305 nt|1| | | |101 AA|101+10=111| |rev-compl|
DIRLTQSPGLLAVSPGEMVIINCKSSQSLLNTFDNKNYFAWH*QKPGQAPKMLISWA...
....STQAPGVP.DQFIGSR..SETDYTLTFSSLQAGDVAVYYCQHYYSTS
>NW_001099007|IGKV4-3*01|Macaca mulatta|F|V-REGION| |305 nt|1| | ||101 AA|101+10=111| | |
DIVMTQSPDSLAVSLGERVTINCKSSQSLLYSSNNKNYLAWYQQKPGQAPKLLIYWA...
....STRESGVP.NRFSGSG..SGSDFTLTISGLQAEDVAVYYCQQYYSTP
>NW_001099007|IGKV5-1*01|Macaca mulatta|F|V-REGION| |287 nt|1| | ||95 AA|95+16=111| | |
ETILTQSAAFVSATPGDKVTISCRAGQDI......DDDMNWYQQEPGEAPKLIIKDA...
....TTLVSGIP.PRFSGSG..YGTDFTLTINNVESEDAAYYFCLQHDNFP
>NW_001099007|IGKV5-2*01|Macaca mulatta|F|V-REGION| |287 nt|1| | ||95 AA|95+16=111| | |
EMILTQSPAFVSATPGDKVTISCRAGQDI......DDDMNWYQQEPGEAPKLIIKDA...
....TTLVSGIP.PQFSGSG..YGIDFTLTINSMKSEDTAYYFCQQRDNTP
>NW_001099007|IGKV6-1*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
EIVLTQSPAFRSVTLKEKVTITCQASQSI......GSSLHWYQQKPDQSPKLLIKYA...
....SQSISGVP.SRFSGSG..SGTDFTLTINSLEAEDAATYYCQQSSSFP
>NW_001099007|IGKV6-2*01|Macaca mulatta|F|V-REGION| |287 nt|1| | | |95 AA|95+16=111| |rev-compl|
DIVMTQSPAFVSVTPGEKVTITCQASEGI......SNYLHWYQQKPDQAPKLFIQYA...
....SQSISGVP.SRFTGSG..SGTDFTFTISSLEVEDAATYYCQQGNKHP
>NW_001099007|IGKV7-1*01|Macaca mulatta|F|V-REGION| |299 nt|1| | | |99 AA|99+12=111| |rev-compl|
DIVLTQSPASLAVSPGQRATITCRASETVSF..FGINLIHWYQQKPGQPPKLLIYQA...
....SNKDTGVP.ARFSGSG..SGTDFTLTINPVEADDAADYYCLQSKNSP

</pre>
<hr />





<a href="/genedb/doc">-> IMGT/GENE-DB Documentation</a><br />
<a href="/genedb/">-> IMGT/GENE-DB Query page</a><br />
<a href="/genedb/directlinks">-> IMGT/GENE-DB direct links</a><br />
<a href="/genedb/stats">-> IMGT/GENE-DB statistics</a><br />
<a href="/IMGTgenedbdoc/programversions.html">-> IMGT/GENE-DB program versions</a><br />
<a href="/IMGTgenedbdoc/dataupdates.html">-> IMGT/GENE-DB data updates</a>

<hr />

<p class="menu">
<a href="/">IMGT Home page</a>
| <a href="/IMGTrepertoire/">IMGT Repertoire (IG and TR)</a>
| <a href="/IMGTrepertoireMH/">IMGT Repertoire (MH)</a>
| <a href="/IMGTrepertoireRPI/">IMGT Repertoire (RPI)</a>
| <a href="/IMGTindex/">IMGT Index</a>
| <a href="/IMGTScientificChart/">IMGT Scientific chart</a>
| <a href="/IMGTeducation/">IMGT Education</a>
</p>



<span class="logosFoot">
<a href="https://www.cnrs.fr"><img src="/images/cnrs.png" alt="logo CNRS" style="vertical-align:middle;" /></a>
<a href="https://www.umontpellier.fr/"><img src="/images/logos/universite_mtp.png" alt="Universit&#233; de Montpellier" style="width:60px;vertical-align:middle;" /></a>
<a href="https://ec.europa.eu/info/research-and-innovation_en"><img src="/images/UE.png" alt="logo UE" style="vertical-align:middle;" /></a>
</span>



<p>&#169;  Copyright 1995-2020 IMGT&reg;, the international ImMunoGeneTics information system&reg;
			| <a href="/about/termsofuse.php">Terms of use</a> | <a href="/about/aboutus.php">About us</a> | <a href="mailto:contact-imgt@igh.cnrs.fr">Contact us</a> | <a href="/about/CitingIMGT.php">Citing IMGT</a></p>

<!-- #echo var=$LAST_MODIFIED -->

</body>
</html>

</body>
</html>